src/main/java/org/influxdb/impl/TimeUtil.java

```diff
@@ -60,7 +60,7 @@ public static String toTimePrecision(final TimeUnit t) {
     case MICROSECONDS:
       return "u";
     case NANOSECONDS:
-      return "n";
+      return "ns";
```

Comment:
unrelated change, if this is a problem, please raise another Issue/PR

Reply:
This is not a problem (it works in both cases), but I've notice that the code is not consistent with documentation.